solr/rootfs/etc/s6-overlay/s6-rc.d/solr/run

```diff
@@ -4,7 +4,7 @@ set -e
 ARGS=(-m "${SOLR_MEMORY}")
 
 if [[ -n "${SOLR_JETTY_OPTS}" ]]; then
-  ARGS+=(-j "${SOLR_JETTY_OPTS}")
+  ARGS+=(--jvm-opts "${SOLR_JETTY_OPTS}")
```

Comment:
not sure when solr changed this; must have been in 9.8 bump. Using `-j` solr would fail to start with 

```
 ERROR: Jetty config is required when using the -j option!
 Usage: solr start [-f] [-c] [--host host] [-p port] [--server-dir directory] [-z zkHost] [-m memory] [-e example] [--solr-home solr.solr.home] [--data-home solr.data.home] [--jvm-opts "jvm-opts"] [-V]
 ```